sass/directives/_boxes.scss

```diff
@@ -51,12 +51,19 @@
   position: relative;
   margin: $base-spacing-small;
   text-decoration: none;
+  font-weight: $font-weight-normal;
+
+  &:hover {
+    box-shadow: $base-box-shadow-hover;
+  }
 
   &:hover,
   &:active,
-  &:focus {
+  &:focus,
+  a:hover,
```

Comment:
This change stemmed from my struggle getting Magnific popups to work with our card structure - it's definitely no longer necessary. Good catch!